### DISCUSSION TITLE
How to make Frame REALLY transparent?

### DISCUSSION THREAD

**Discussion by roberts-erglis** [TomSchimansky/CustomTkinter#2190] [Q&A]
With **fg_color** and **bg_color** don't actually make invisible for frame that are front of other frame. Searched everywhere in documentation and still no. I believe it may do with `.configure()` but there is no documentation for it. Also with **theme.json** still can't do this.

**Part of code:**
```
#[Background]
main_fr = CTkLabel(root, corner_radius=0, image=_bg_green, text=" ")
main_fr.place(relwidth = 1, relheight=1)

#[Top menu bar]
top_fr = CTkFrame(root, corner_radius=0, fg_color=_t, bg_color=_t)
top_fr.place(relheight=0.2, relwidth=1)

enter_bt_frame = CTkButton(top_fr, corner_radius=0, text=" ", fg_color=_t, bg_color=_t, hover=False)
enter_bt_frame.place(anchor="n", relx=0.5, relheight=0.8, relwidth=1)

leave_bt_frame = CTkButton(top_fr, corner_radius=0, text=" ", fg_color=_t, hover=False)
leave_bt_frame.place(anchor="s", relx=0.5, rely=1, relheight=0.2, relwidth=1)
```

![image](https://github.com/TomSchimansky/CustomTkinter/assets/60196003/7e1f9e8b-5641-4293-b673-66ee6d398b77)

**Comment by Anonymous6598** [comment]
just put `bg_color=transparent` and it will be transparent.

**Reply by Anonymous6598** [reply]
```python
top_fr = CTkFrame(top_fr, corner_radius=0, fg_color="transparent", bg_color="transparent") #bg_color is used for outer part and fg_color is used for inner
``` 
there you have it

**Reply by roberts-erglis** [reply]
As I stated fg_color and bg_color don't actually did it job with "transparent". I saw that I didn't added specific about with "transparent" in problem. Thanks for trying.

**Reply by Anonymous6598** [reply]
You're welcome
